api/cmd/managerapi.go

```diff
@@ -69,85 +69,102 @@ func NewManagerAPICommand() *cobra.Command {
 				os.Exit(0)
 			}
 
-			conf.InitConf()
-			log.InitLogger()
-
-			if err := utils.WritePID(conf.PIDPath); err != nil {
-				log.Errorf("failed to write pid: %s", err)
-				panic(err)
-			}
-			utils.AppendToClosers(func() error {
-				if err := os.Remove(conf.PIDPath); err != nil {
-					log.Errorf("failed to remove pid path: %s", err)
-					return err
-				}
-				return nil
-			})
-
-			droplet.Option.Orchestrator = func(mws []droplet.Middleware) []droplet.Middleware {
-				var newMws []droplet.Middleware
-				// default middleware order: resp_reshape, auto_input, traffic_log
-				// We should put err_transform at second to catch all error
-				newMws = append(newMws, mws[0], &handler.ErrorTransformMiddleware{}, &filter.AuthenticationMiddleware{})
-				newMws = append(newMws, mws[1:]...)
-				return newMws
+			service, err := createService()
+			if err != nil {
+				return err
 			}
+			status, err := service.manageService()
+			fmt.Printf("%s\n", status)
```

Comment:
Debug info ?

Reply:
Hii, that's the formatted output after executing any commands ( the green  [  OK  ])